src/main/java/com/google/sps/servlets/ImageHandlerServlet.java

```diff
@@ -31,108 +32,99 @@
   private static final String CURRENT_PLAYER_TRUE_PARAMETER = "true";
   private static final String CURRENT_PLAYER_FALSE_PARAMETER = "false";
   private static final String DISPLAY_NAME_PARAMETER = "displayName";
+  private static final String MESSAGE_PARAMETER = "message";
   private static final String EMAIL_PARAMETER = "email";
+  private static final String EMPTY_PARAMETER = "";
   private static final String ID_PARAMETER = "id";
+  private static final String IMAGE_ID_PARAMETER = "imageID";
   private static final String IMAGE_PARAMETER = "image";
-  private static final String JPEG_PARAMETER = ".jpeg";
-  private static final String JPG_PARAMETER = ".jpg";
+  private static final String DEFAULT_PARAMETER = "default";
   private static final String PLAYER_QUERY_PARAMETER = "Player";
-  private static final String PNG_PARAMETER = ".png";
   private static final String UPLOADED_REDIRECT_PARAMETER = "/uploaded.html";
   private static final String LOGIN_REDIRECT_PARAMETER = "/userAuthPage.html";
+  private static final String CHARACTER_DESIGN_REDIRECT_PARAMETER = "/characterDesign.html";
+  private static final String CONTENT_TYPE = "text/html";
+  private static final User USER = UserServiceFactory.getUserService().getCurrentUser();
   private static final String DEFAULT_IMAGE_GS_LOCATION =
       "/gs/cs-career-step-2020.appspot.com/face.jpg";
-  private static Entity currentPlayer = currentPlayer();
-  private static boolean isLoggedIn = UserServiceFactory.getUserService().isUserLoggedIn();
+  private Entity player;
+  private static final DatastoreService DATASTORE = DatastoreServiceFactory.getDatastoreService();
 
-  private static Entity currentPlayer() {
-    Entity entity = new Entity("Player");
-    try {
-      entity = PlayerDatabase.getCurrentPlayerEntity();
-    } catch (Exception e) {
-    }
-    return entity;
-  }
+  private static boolean isLoggedIn = UserServiceFactory.getUserService().isUserLoggedIn();
```

Comment:
Does this being static every introduce any unwanted behavior? Like what if a user signs out at a later timepoint?